qiskit/circuit/library/standard_gates/y.py

```diff
@@ -123,7 +123,15 @@ def control(
         return gate
 
     def inverse(self, annotated: bool = False):
-        r"""Return inverted Y gate (:math:`Y^{\dagger} = Y`)"""
+        r"""Return inverted Y gate (:math:`Y^{\dagger} = Y`)
+
+        Args:
+            annotated: indicates whether the inverse gate can be implemented
+                as an annotated gate.
```

Comment:
```suggestion
            annotated: Typically when set to ``True`` this is used to return an :class:`.AnnotatedOperation` 
                with an inverse modifier set instead of a concrete :class:`.Gate`. However, for this class this
                argument is ignored as this gate is self inverse.
```